artifactory/commands/transferconfig/configxmlutils/federatedrepos.go

```diff
@@ -21,87 +15,47 @@ func ReplaceUrlsInFederatedrepos(configXml, sourceBaseUrl, targetBaseUrl string)
 	}
 	results := ""
 	prefix, content, suffix := splitXmlTag(configXml, xmlTagIndices, 0)
-	federatedRepositories, err := fixFederatedRepositories(content, sourceBaseUrl, targetBaseUrl)
+	federatedRepositories, err := fixFederatedRepositories(content)
 	if err != nil {
 		return "", err
 	}
 	results += prefix + federatedRepositories + suffix
 	return results, nil
 }
 
-func fixFederatedRepositories(content, sourceBaseUrl, targetBaseUrl string) (string, error) {
+func fixFederatedRepositories(content string) (string, error) {
 	xmlTagIndices, exist, err := findAllXmlTagIndices(content, `federatedRepository`, false)
 	if err != nil {
 		return "", err
 	}
 	if !exist {
 		return content, nil
 	}
+	federatedMembersRemoved := false
 	results := ""
 	for i := range xmlTagIndices {
 		prefix, content, suffix := splitXmlTag(content, xmlTagIndices, i)
-		federatedRepository, err := fixFederatedRepository(content, sourceBaseUrl, targetBaseUrl)
+		federatedRepository, removed, err := removeFederatedMembers(content)
 		if err != nil {
 			return "", err
 		}
 		results += prefix + federatedRepository + suffix
+		federatedMembersRemoved = federatedMembersRemoved || removed
```

Comment:
Let's describe this line, and specifically why both `federatedMembersRemoved` and `removed` are used.